include/ctr/gte.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <ctr/math.h>
-#include <psn00bsdk/include/inline_c.h>
+#include <ctr/nugget/inline_n.h>
```

Comment:
Are you more sure that this doesn't have bugs in it like psn00bsdk? If it does have bugs, will we have to manually vet gte instructions ourselves?

Reply:
This one doesn't, yea